org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/ScanProcess.java

```diff
@@ -277,30 +277,30 @@ private void setFilePath(ScanBean bean) throws EventException {
 			bean.setFilePath(req.getFilePath());
 		}
 		logger.debug("Nexus file path set to {}", bean.getFilePath());
-		
+
 	}
-	
+
 	/**
 	 * Initialise the malcolm device with the point generator and the malcolm model
 	 * with its output directory. This needs to be done before validation as these values
-	 * are sent to the actual malcolm device over the connection for validation. 
+	 * are sent to the actual malcolm device over the connection for validation.
 	 * @param gen
 	 * @throws EventException
 	 * @throws ScanningException
 	 */
 	private void initializeMalcolmDevice(ScanBean bean, IPointGenerator<?> gen) throws EventException, ScanningException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.eclipse.scanning.api.event.EventException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)